package.json

```diff
@@ -42,8 +43,9 @@
     "react-redux": "^7.2.0",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.1",
-    "serverless": "^2.29.0",
-    "serverless-finch": "^2.6.0",
+    "serverless": "^3.20.0",
+    "serverless-finch": "^4.0.0",
+    "serverless-single-page-app-plugin": "file:./.serverless_plugins/serverless-single-page-app-plugin",
```

Comment:
You can put file with plugin inside .serverless_plugins folder and then specify this plugin at plugins section in serverless.yml

<img width="781" alt="Screenshot 2022-07-12 at 18 16 54" src="https://user-images.githubusercontent.com/32521973/178526445-b4ce9428-0f1a-496e-a82a-ef1082cb1032.png">